docs/code-signing.rst

```diff
@@ -0,0 +1,10 @@
+Code signing policy
+===================
+
+The code used by the DebOps Project and commited in its repositories is
+expected to be signed by a valid GPG key of a DebOps Developer or DebOps
+Contributor. This should allow for secure code authentication and a valid audit
```

Comment:
What does "is expected to be signed" mean? It must? Or it should? What if it isn't?

For my taste this is a bit exaggerated. As there is no controllable deployment mechanism (only git pull) or stable APIs (variable naming, sematics), I'm not really worried about someone bringing in unauthorized changes, but more about someone bringing in changes which will blow my setup. Therefore I need to do a manual code audit anyway before rolling out any changes to my environment. Signed commits won't save me from this. If it's optional, I'm fine with it.

I see that it could make sense for DebOps core developers which have full commit privileges on all repositories. But even there, if your laptop with your unprotected SSH private key and your GPG key is gone, even a commit signature won't protect the code. Or did I miss something? Maybe you should add a link somewhere why code signing makes sense in the first place. What happens if you revoke the key that was used for signing?